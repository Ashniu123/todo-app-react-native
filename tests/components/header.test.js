import React from 'react';
import { shallow } from 'enzyme';

import Header from '../../src/components/header/header.component';

test('should render Header component correctly', () => {
	const wrapper = shallow(<Header />);

	expect(wrapper).toMatchSnapshot();
});