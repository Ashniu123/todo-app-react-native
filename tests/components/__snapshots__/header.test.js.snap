// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header component correctly 1`] = `
<Component
  style={
    Object {
      "backgroundColor": "#42A5F5",
      "height": 40,
      "justifyContent": "center",
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "alignSelf": "center",
        "color": "white",
        "fontSize": 20,
      }
    }
  >
    Todo List
  </Text>
</Component>
`;
